src/action.ts

```diff
@@ -14,6 +14,8 @@ import { pipeline as streamPipeline } from "node:stream/promises"
 import AdmZip from "adm-zip"
 import { Readable } from "stream"
 import { randomUUID } from "crypto"
+import { mkdtemp } from "fs/promises"
```

Comment:
You don't need to import it. It's already available through `fs`, which is declared above. Also, note that we want to use the synchronous form: `mkdtempSync`.